packages/vuetify/src/components/VAlert/VAlert.ts

```diff
@@ -168,13 +168,16 @@ export default mixins(
       return this.hasColoredIcon ? this.computedColor : undefined
     },
     isDark (): boolean {
-      if (
-        this.type &&
-        !this.coloredBorder &&
-        !this.outlined
-      ) return true
-
-      return Themeable.options.computed.isDark.call(this)
+      if (this.dark === true) {
```

Comment:
This is exactly the same as the themeable mixin that `v-alert` already extends. https://github.com/vuetifyjs/vuetify/blob/b1c3f81bb62bc218b1c18236f7252412722b32fc/packages/vuetify/src/mixins/themeable/index.ts#L51